### PR TITLE
Reenables miscreants on the RP server with some better-curated objectives.

### PR DESCRIPTION
**Pull request by F-OS**
[INPUT WANTED] [FEATURE]
## About the PR

This PR enables miscreants on the RP-Server. Objectives courtesy of Gonzako, Jan Antilles, Flap Jackson, schwick, and some others I'm probably forgetting.

## Why's this needed? 

I think that having miscreants enabled on the RP server might lead to better stories, give people some more direction, and improve rounds. I do see the problems with regular miscreant objectives on the RP server, and I hope these seem non-grief-enabling enough for everyone to be comfortable with em, if not, feel free to either suggest new ones or just ask me to cut em out entirely!

Also if the writing looks bad I'm sorry grammar isn't my strong-suit.

## Changelog

```
(u)Fosstar:
(*) Miscreants are now reenabled on the RP server with new objectives. Be wary of these fiends, slinking around to cause all manner of trouble.
```

### CODE REVIEW

File: code/datums/miscreant_objectives.dm [hunk 1]
```diff
@@ -119,4 +122,62 @@ ABSTRACT_TYPE(/datum/objective/miscreant)
...
+		explanation_text = "Attempt to convince the crew that everything was better in the old days. Try to convert as many humans into monkeys and stray away from any advanced tech."
+
+	narc
+		explanation_text = "Forge evidence and bribe people into testifying that an entire department is involved in the illegal drug trade with the goal of convincing security to raid that department."
+
+	water
+		explanation_text = "Become an extremely obnoxious hydration advocate. Constantly remind people to drink lots of water."
+
+	missionary
+		explanation_text = "found a new religion and be as obnoxious as you can about spreading said religion. go door-to-door between apartments and attempt to convince people to convert."
```
Comment (geneva-breaker) [inline code/datums/miscreant_objectives.dm:179]: capitalisation

File: code/datums/miscreant_objectives.dm [hunk 2]
```diff
@@ -119,4 +122,62 @@ ABSTRACT_TYPE(/datum/objective/miscreant)
...
+		explanation_text = "Destroy as many shrubs as you can. Replace the destroyed shrubs with other objects. If caught, claim to be balancing the Feng-Shui of the station."
+
+	detective
+		explanation_text = "Found a private detective agency and attempt to solve cases before the detective can. Come up with extremely absurd explanations for crimes and insist that security is secretly in on it."
+
+	exterminator
+		explanation_text = "Kill as many non-monkey, non-pet animals aboard the station as possible and bring their corpses to the bridge. Once finished, claim to be a trained exterminator and demand payment for your services."
+
+	conspiracy
+		explanation_text = "Spin wild theories about how every single accident or problem aboard the station is the fault of some shadowy NT/syndicate cabal. Attempt to get on the radio show and share your theories with the world."
```
Comment (geneva-breaker) [inline code/datums/miscreant_objectives.dm:167]: this has the potential to get really annoying and repetitive. people do this gimmick often enough without being *asked* to do it.

File: code/datums/miscreant_objectives.dm [hunk 3]
```diff
@@ -119,4 +122,62 @@ ABSTRACT_TYPE(/datum/objective/miscreant)
...
+
+	primitivism
+		explanation_text = "Attempt to convince the crew that everything was better in the old days. Try to convert as many humans into monkeys and stray away from any advanced tech."
```
Comment (geneva-breaker) [inline code/datums/miscreant_objectives.dm:170]: last sentence syntactically dubious

File: code/datums/miscreant_objectives.dm [hunk 4]
```diff
@@ -119,4 +122,62 @@ ABSTRACT_TYPE(/datum/objective/miscreant)
 	spy
 		explanation_text = "Become an inter-galactic spy and refer to everyday objects as 'gadgets'."
 
+ABSTRACT_TYPE(/datum/objective/miscreantrp)
+/datum/objective/miscreantrp
+	shoes
+		explanation_text = "Through trade, sweet-talking, light extortion, blackmail, and other devilish, but non-violent means, secure and deep-fry as many shoes as possible"
```
Comment (geneva-breaker) [inline code/datums/miscreant_objectives.dm:128]: great

**Review by geneva-breaker — CHANGES_REQUESTED** [review]
really good, just a few comments

File: code/datums/miscreant_objectives.dm (hunk 4 above)
Comment (F-OS) [reply code/datums/miscreant_objectives.dm]: Thanks!

File: code/datums/miscreant_objectives.dm (hunk 2 above)
Comment (F-OS) [reply code/datums/miscreant_objectives.dm]: removed and replaced with a safer objective from the regular miscreant pool.

File: code/datums/miscreant_objectives.dm (hunk 1 above)
Comment (F-OS) [reply code/datums/miscreant_objectives.dm]: fixed

File: code/datums/miscreant_objectives.dm (hunk 3 above)
Comment (F-OS) [reply code/datums/miscreant_objectives.dm]: fixed in new commit

**Review by ZeWaka — APPROVED** [review]

**Pull request merged**